README.md

```diff
@@ -24,3 +24,40 @@ Then:
     $ source ppenv/bin/activate
     $ pip install -r requirements.txt
     $ python setup.py develop
+
+## Creating an AWS Lambda Zipfile
+
+After modifying ``push_processor/settings.js`` for your appropriate settings, a
+Lambda-ready zipfile can be created with make:
+
+    $ make lambda-package
+
+Ensure that no extra files are present in the work directory, as all files in
+the work directory are included except files listed in the ``ignore`` section
+of ``lambda.json``.
+
+The AWS handler you should set is: ``lambda.handler``, which will then call
+the Push Processor.
+
+### Lambda VPC Access
+
+The Push Processor needs access to both Redis and AWS DynamoDB, as such it will
```

Comment:
"The Push Processor needs access to both Redis and AWS DynamoDB. It will likely be necessary..."